opal/mca/memory/patcher/configure.m4

```diff
@@ -38,53 +38,18 @@ AC_DEFUN([MCA_opal_memory_patcher_CONFIG],[
 
     OPAL_VAR_SCOPE_PUSH([memory_patcher_have___curbrk memory_patcher_have___mmap memory_patcher_have___syscall memory_patcher_have___mmap_prototype memory_patcher_have___syscall_prototype])
```

Comment:
Should probably remove `memory_patcher_have___syscall` (and friends) from this `OPAL_VAR_SCOPE`, too.